--- ffmpeg/libavfilter/Makefile	2023-11-11 00:25:17.000000000 +0000
+++ ffmpeg/libavfilter/Makefile	2024-08-05 20:40:52.264549503 +0000
@@ -6,6 +6,7 @@
           buffersrc.h                                                   \
           version.h                                                     \
           version_major.h                                               \
+          drawutils.h                                                   \
 
 OBJS = allfilters.o                                                     \
        audio.o                                                          \
